### PR TITLE
Update vision server to use UDP

### PR DESCRIPTION
**Pull request by WasabiFan**

### CODE REVIEW

File: CompetitionTemplate/src/competition/subsystems/vision/commands/SpewVisionInformationCommand.java [hunk 1]
```diff
@@ -39,7 +39,7 @@ public void execute() {
     
     public void updateSpewInfo() {
         int numSpatial = visionSubsystem.getBoulderInfo() == null ? 0 : visionSubsystem.getBoulderInfo().length;
-        log.debug("Currently has " + numSpatial + " spatial coords"
+        log.info("Currently has " + numSpatial + " spatial coords"
```
Comment (aschokking) [inline CompetitionTemplate/src/competition/subsystems/vision/commands/SpewVisionInformationCommand.java]: How often does this get run?

Comment (WasabiFan) [inline CompetitionTemplate/src/competition/subsystems/vision/commands/SpewVisionInformationCommand.java]: Every 10 seconds

**Comment by JohnGilb** [comment]
Is this the code where you're seeing memory leaks?

**Comment by WasabiFan** [comment]
Sort of? I don't know what is leaking. Whatever the case, the suspect code is already in master, so it isn't a blocking issue for this PR. If you think it is an issue that deserves more immediate attention, I can find a heap dissector tool and see what is taking up memory.

**Pull request merged**